src/core/surface/init.c

```diff
@@ -64,7 +64,7 @@ void grpc_init(void) {
     grpc_security_pre_init();
     grpc_iomgr_init();
     grpc_tracer_init("GRPC_TRACE");
-    census_init();
+    census_initialize(CENSUS_NONE);
```

Comment:
Looks like other components' init functions are all named xxx_init() instead of xxx_initialize().